docker/docker-compose.yml

```diff
@@ -453,5 +453,22 @@ services:
       aws_cli:
         condition: service_healthy
 
+  camunda-tools:
+    healthcheck:
+      test: ["CMD", "java", "/app/scripts/HealthCheck.java", "http://localhost:8080/actuator/health", "||", "exit", "1"]
+      start_period: 1m
+      timeout: 30s
+      retries: 3
+    image: quay.io/ukhomeofficedigital/hocs-camunda-tools:0.0.1
+    ports:
+      - 8089:8080
```

Comment:
`8089` is actually the default port that hocs-templates uses and is part of the following #12 to put it into the docker-compose.

Reply:
Good spot, I'll move ports